tests/test-mapping.php

```diff
@@ -4,62 +4,62 @@
  * @group mapping
  */
 class Tests_Mapping extends SearchPress_UnitTestCase {
-	var $demo_user;
-	var $demo_user_id;
-	var $demo_term;
-	var $demo_term_id;
-	var $demo_post;
-	var $demo_post_id;
-	var $demo_dates = array();
-
-	function setUp() {
-		parent::setUp();
-
-		$this->demo_user = array(
-			'user_login' => 'author1',
+	protected static $demo_user;
```

Comment:
Would be useful to have comments for each of these, including indicating their data type, for better readability and comprehension of the test suite.